include/modem/location.h

```diff
@@ -478,20 +496,42 @@ int location_pgps_data_process(const char *buf, size_t buf_len);
  * @details If the Location library is not receiving cellular position directly from services,
  * it triggers the @ref LOCATION_EVT_CELLULAR_EXT_REQUEST event when neighbor cell information
  * should be sent to cloud services for location resolution. Then, the application responds
- * with the result of the location result.
+ * with the result.
  *
- * In addition to successful and error results, the application can indicate an unknown result with
- * @ref LOCATION_CELLULAR_EXT_RESULT_UNKNOWN. This is useful when the application wants
+ * In addition to 'success' and 'error' results, the application can indicate that the result is
+ * unknown with @ref LOCATION_EXT_RESULT_UNKNOWN. This is useful when the application wants
  * the Location library to proceed irrespective of the outcome. The Location library will try to
  * perform a fallback to the next method, if available, just like in a failure case.
  * If there are no more methods, LOCATION_EVT_RESULT_UNKNOWN event will be sent to the application.
  *
  * @param[in] result Result of the external cellular request.
  * @param[in] location Cellular location data. Will be used only if @p result is
- *                     LOCATION_CELLULAR_EXT_RESULT_SUCCESS.
+ *                     LOCATION_EXT_RESULT_SUCCESS.
  */
 void location_cellular_ext_result_set(
-	enum location_cellular_ext_result result,
+	enum location_ext_result result,
+	struct location_data *location);
+
+/**
+ * @brief Pass Wi-Fi location result to the library.
+ *
+ * @details If the Location library is not receiving Wi-Fi position directly from services,
+ * it triggers the @ref LOCATION_EVT_WIFI_EXT_REQUEST event when access point information
+ * should be sent to cloud services for location resolution. Then, the application responds
```

Comment:
```suggestion
 * @details If the Location library is not receiving Wi-Fi position directly from the services,
 * it triggers the @ref LOCATION_EVT_WIFI_EXT_REQUEST event when access point information
 * should be sent to cloud services for location resolution. Then, the application responds
```
Not that sure I understand what comes after the event. Does that event tell the exact time (when) the AP info should be sent? Or does it rather mean that as a result of the event, the AP info is sent?

Reply:
The event indicates that here is the AP info, you can send it to cloud now. Or obviously application can do whatever it wants to but in practice it'll send the list to some cloud service for location resolution.

Reply:
Maybe then something like: "event, that indicates the access point information is ready/available to be sent to...."

Reply:
Modified. Please check.